pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -46,6 +46,10 @@ message Empty {
 
 // DevicePlugin is the service advertised by Device Plugins
 service DevicePlugin {
+	// GetDevicePluginOptions returns options to be communicated with Device
+        // Manager
+	rpc GetDevicePluginOptions(Empty) returns (DevicePluginOptions) {}
```

Comment:
`GetDevicePluginOptionsRequest` and `GetDevicePluginOptionsResponse` ?

Reply:
I was wondering about this but feel the chance we need to send extra information through this rpc seems quite low.